CHANGELOG.adoc

```diff
@@ -1,11 +1,22 @@
-== iZettle API Changelog
+== Zettle API Changelog
+.**April 30, 2021** `Pusher API`
+[%collapsible]
+====
+**Pusher API documentation Improvement**
+
+The Pusher API documentation is improved as follows:
```

Comment:
Lets say "The improvements include the following" since the title already talks of it.